lib/coinbase/address.rb

```diff
@@ -62,6 +62,27 @@ def balance(asset_id)
       Coinbase::Balance.from_model_and_asset_id(response, asset_id).amount
     end
 
+    # Enumerates the historical balances for a given asset belonging of address.
+    # The result is an enumerator that lazily fetches from the server, and can be iterated over,
+    # converted to an array, etc...
+    # @return [Enumerable<Coinbase::HistoricalBalance>] Enumerator that returns historical_balance
+    def hiatorical_balances(asset_id)
+      Coinbase::Pagination.enumerate(
+        ->(page) { list_page(asset_id, page) }
+      ) do |historical_balance|
+        Coinbase::HistoricalBalance.from_model(historical_balance)
+      end
+    end
+
+    def list_page(asset_id, page)
```

Comment:
This should be beneath the `private` directive

Reply:
thanks for catching! let me move it.